rand_distr/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+- Improve algorithm for sampling `Beta` (#1000)
```

Comment:
Could this be a *little* more specific?

Reply:
I changed it to "New `Beta` sampling algorithm for improved performance and accuracy".